diff -up ./fizz/CMakeLists.txt.orig ./fizz/CMakeLists.txt
--- ./fizz/CMakeLists.txt.orig
+++ ./fizz/CMakeLists.txt
@@ -265,7 +265,8 @@
     ${FIZZ_LINK_LIBRARIES}
     ${DOUBLE_CONVERSION_LIBRARIES}
     ${CMAKE_DL_LIBS}
-    ${LIBRT_LIBRARIES})
+    ${LIBRT_LIBRARIES}
+    execinfo)
 
 if ($FIZZ_SHINY_DEPENDENCIES)
   add_dependencies(fizz ${FIZZ_SHINY_DEPENDENCIES})
@@ -336,6 +337,7 @@
     fizz
     ${LIBGMOCK_LIBRARIES}
     ${GLOG_LIBRARY}
+    execinfo
 )
 
 target_compile_definitions(fizz_test_support
@@ -379,7 +381,8 @@
     ${test_name}
     fizz
     fizz_test_support
-    ${LIBGMOCK_LIBRARIES})
+    ${LIBGMOCK_LIBRARIES}
+    execinfo)
 
   if(NOT ${CMAKE_CXX_COMPILER_ID} STREQUAL MSVC)
     # GMOCK_MOCK_METHOD() will complain otherwise
@@ -465,7 +468,7 @@
 
 if(BUILD_EXAMPLES)
   add_executable(BogoShim test/BogoShim.cpp)
-  target_link_libraries(BogoShim fizz sodium)
+  target_link_libraries(BogoShim fizz sodium execinfo)
   set_target_properties(BogoShim PROPERTIES OUTPUT_NAME fizz-bogoshim)
   install(
     TARGETS BogoShim
@@ -479,7 +482,7 @@
       tool/FizzGenerateDelegatedCredentialCommand.cpp
       tool/FizzServerBenchmarkCommand.cpp
       tool/FizzServerCommand.cpp)
-  target_link_libraries(FizzTool fizz sodium)
+  target_link_libraries(FizzTool fizz sodium execinfo)
   set_target_properties(FizzTool PROPERTIES OUTPUT_NAME fizz)
   install(
     TARGETS FizzTool
