diff -up ./fizz/CMakeLists.txt.orig ./fizz/CMakeLists.txt
--- ./fizz/CMakeLists.txt.orig
+++ ./fizz/CMakeLists.txt
@@ -252,7 +252,8 @@
     ${FIZZ_LINK_LIBRARIES}
     ${DOUBLE_CONVERSION_LIBRARIES}
     ${CMAKE_DL_LIBS}
-    ${LIBRT_LIBRARIES})
+    ${LIBRT_LIBRARIES}
+    execinfo)
 
 if ($FIZZ_SHINY_DEPENDENCIES)
   add_dependencies(fizz ${FIZZ_SHINY_DEPENDENCIES})
@@ -323,6 +324,7 @@
     fizz
     ${LIBGMOCK_LIBRARIES}
     ${GLOG_LIBRARY}
+    execinfo
 )
 
 target_compile_definitions(fizz_test_support
@@ -366,7 +368,8 @@
     ${test_name}
     fizz
     fizz_test_support
-    ${LIBGMOCK_LIBRARIES})
+    ${LIBGMOCK_LIBRARIES}
+    execinfo)
 
   if(NOT ${CMAKE_CXX_COMPILER_ID} STREQUAL MSVC)
     # GMOCK_MOCK_METHOD() will complain otherwise
@@ -452,7 +455,7 @@
 
 if(BUILD_EXAMPLES)
   add_executable(BogoShim test/BogoShim.cpp)
-  target_link_libraries(BogoShim fizz sodium)
+  target_link_libraries(BogoShim fizz sodium execinfo)
   add_executable(FizzTool
       tool/Main.cpp
       tool/FizzClientCommand.cpp
@@ -461,6 +464,6 @@
       tool/FizzGenerateDelegatedCredentialCommand.cpp
       tool/FizzServerBenchmarkCommand.cpp
       tool/FizzServerCommand.cpp)
-  target_link_libraries(FizzTool fizz sodium)
+  target_link_libraries(FizzTool fizz sodium execinfo)
   set_target_properties(FizzTool PROPERTIES OUTPUT_NAME fizz)
 endif()
