diff -up ./src/py_proc_list.c.orig ./src/py_proc_list.c
--- ./src/py_proc_list.c.orig
+++ ./src/py_proc_list.c
@@ -218,7 +218,7 @@
   for (;;) {
     // This code is inspired by the ps util
     ent = readdir(proc_dir);
-    if (!ent || !ent->d_name) break;
+    if (!ent) break;
     if ((*ent->d_name <= '0') || (*ent->d_name > '9')) continue;
 
     unsigned long pid = strtoul(ent->d_name, NULL, 10);
